hamming/gigasecond.rb

```diff
@@ -0,0 +1,5 @@
+class Gigasecond
+	def self.from(t)
+	 	t+10000000000
+	end
```

Comment:
Use proper indentation. Give proper name to this number and move to variable. Add space before and after `+`